src/Event/Subscriber/MutationTestingConsoleLoggerSubscriber.php

```diff
@@ -184,14 +184,38 @@ private function showMetrics(): void
         $this->output->writeln(['', '']);
         $this->output->writeln('<options=bold>' . $this->metricsCalculator->getTotalMutantsCount() . '</options=bold> mutations were generated:');
         $this->output->writeln('<options=bold>' . $this->getPadded($this->metricsCalculator->getKilledByTestsCount()) . '</options=bold> mutants were killed by Test Framework');
```

Comment:
these 2 metrics are the most important ones, which I think should always be visible